src/Mobile/BDMobileValidator.php

```diff
@@ -47,4 +48,15 @@ public static function getValidated(string $number): string
 
         return BDMobileFormatter::format($number);
     }
+
+    /**
+     * @param string $number
+     * @return bool
+     */
+    public static function isContainNumberWithoutBDCountryCode(string $number) : bool
+    {
+        $number = preg_replace('/^(\+88)/', '', $number);
+        if (preg_match("/^0[0-9]*$/",$number)) return  true;
```

Comment:
If statement can be removed, as the condition itself returns true/false.
https://adamj.eu/tech/2020/01/17/simplify-your-ifs-that-return-booleans/